.github/workflows/check-for-non-releasable-actions.yaml

```diff
@@ -0,0 +1,78 @@
+name: Check for non-releasable actions
+on:
+  pull_request:
+    paths:
+      - actions/
```

Comment:
Yeah, I believe this is all we need. Also added `release-please-config.json` since in case we removed a package from there it's not going to run the check.